src/main/java/objects/ProcessingTool.java

```diff
@@ -1,12 +1,26 @@
 package objects;
 
-import java.util.ArrayList;
-
 //TODO make this class
 public class ProcessingTool {
+    Process[] processes;
```

Comment:
Would prefer the use of ArrayLists as it is easier to iterate through, and more in-keeping with how the rest of the project is structured.